tool/gravity/cli/clusterconfig.go

```diff
@@ -280,7 +281,26 @@ func validateClusterConfig(localEnv *localenv.LocalEnvironment, update libcluste
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	return validate.ClusterConfiguration(existing, update)
+	err = validate.ClusterConfiguration(existing, update)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	server, err := findLocalServer(cluster.ClusterState.Servers)
+	if err != nil {
+		return trace.NotFound("unable to find server info")
```

Comment:
I would make the error more helpful for potential troubleshooting i.e. "unable to find local node among cluster state servers: %v".